hyperon_das/das.py

```diff
@@ -416,18 +416,28 @@ def get_incoming_links(self, atom_handle: str, **kwargs) -> List[Union[Dict[str,
         """
         return self.query_engine.get_incoming_links(atom_handle, **kwargs)
 
-    def count_atoms(self) -> Tuple[int, int]:
+    def count_atoms(self, parameters: Optional[Dict[str, Any]] = None) -> Dict[str, int]:
         """
-        Count nodes and links in DAS.
+        Count Atoms or nodes and links in DAS.
+
+        By default, the precise parameter is set to False returning the total number of atoms, node and link count
```

Comment:
I understood by your comments in the Mattermost channel that you slightly changed this to avoid returning Zeroes to node and link counts when precise is True.

Reply:
Yes, fixed